tasks/oah-update.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Checking latest version
+  shell: 'curl -s https://api.github.com/repos/opensbom-generator/spdx-sbom-generator/releases/latest | grep "browser_download_url.*linux-amd64.tar.gz" | cut -d : -f 2,3 | tr -d \" | head -1 | xargs'
+  register: update_link
+
+- name: Checking installation
+  include: "tasks/oah-validate.yml"
+  
+- name: Checking if latest version is installed
+  debug:
+    msg: "Latest version already installed"
+  when: '"{{update_link.stdout}}" == "{{release_link}}/{{latest}}"'
+
+- name: Updating to latest version

```

Comment:
Error.
![image](https://user-images.githubusercontent.com/58501088/190921439-335f8f5f-21b1-4892-9983-569f68303e55.png)

Reply:
@asa1997 Please install curl in your system. Error shows that it couldn't fetch the source from the link. 
Although I am adding the functionality to check and install curl in install.yml.

Reply:
Added the validation for curl and install it if not present in install.yml (commit : c153afe6a0c10ecde90ce966bb25f26b9d614eda)

Reply:
Error has been resolved.